install.md

```diff
@@ -1,4 +1,4 @@
-# Ubuntu installation instructions
+# Installation instructions.
```

Comment:
I don't think you should change the title, that ubuntu script is still just for ubuntu, https://github.com/onaio/onadata/blob/master/script/install/ubuntu